plugins/woocommerce/src/Admin/Features/Features.php

```diff
@@ -343,6 +343,11 @@ public static function add_admin_body_classes( $admin_body_class = '' ) {
 			$classes[] = sanitize_html_class( 'woocommerce-feature-enabled-' . $feature_key );
 		}
 
+		// Add the Reactify settings payments class if the feature is enabled.
+		if ( FeaturesUtil::feature_is_enabled( 'reactify-classic-payments-settings' ) ) {
+			$classes[] = 'woocommerce-feature-enabled-reactify-classic-payments-settings';
```

Comment:
One nitpick, could we maybe check if this array doesn't already contain the class before adding it? I'm just thinking if in future the class were to get added somewhere else, we could prevent adding it twice.

Reply:
That's a great suggestion. I have added a check if the class already exists.